app/models/release.rb

```diff
@@ -0,0 +1,6 @@
+class Release < ActiveRecord::Base
+  validates :title, presence: true
+  validates :subtitle, presence: true
+  validates :released_on, presence: true
+  validates :body, presence: true
```

Comment:
You can put all these on the same line
`validates :title, :subtitle, :released_on, :body, presence: true`
https://guides.rubyonrails.org/active_record_validations.html#presence